components/UI/Drawer/Drawer.tsx

```diff
@@ -113,107 +156,135 @@ const Drawer = () => {
         role="presentation"
         onKeyDown={(e) => toggler(e)}
       >
-        <div className={styles.content}>
-          <h3 style={{ maxWidth: "45ch" }}>{formatted_address}</h3>
-          <p>{formattedCoordinates}</p>
-          <div className={styles.contentButtons}>
-            {mapsButtons.map((button) => (
-              <Button
-                key={button.label}
-                variant="contained"
-                onClick={() => {
-                  button.urlCallback(
-                    geometry.location.lat,
-                    geometry.location.lng
-                  );
-                }}
-                color={button.color}
-                className={styles.externalLinkButton}
-                startIcon={button.icon}
-              >
-                {button.label}
-              </Button>
-            ))}
-          </div>
-          <div>
-            <TextField
-              fullWidth
-              variant="standard"
-              size="small"
-              value={generateGoogleMapsUrl(
-                geometry.location.lat,
-                geometry.location.lng
-              )}
-              InputProps={{
-                readOnly: true,
-              }}
-            />
-            <div className={styles.actionButtons}>
-              <Button
-                variant="outlined"
-                className={styles.clipboard}
-                size="small"
-                fullWidth
-                onClick={() =>
-                  copyBillboard(
-                    `https://www.google.com/maps/@${geometry.location.lat.toString()},${geometry.location.lng.toString()},22z`
-                  )
-                }
-                startIcon={<CopyAll className={styles.btnIcon} />}
-              >
-                Kopyala
-              </Button>
-              <Button
-                variant="outlined"
-                className={styles.clipboard}
+        {isLoading && (
```

Comment:
Aslinda SWR icin da global loading handler implement edilebiliyor ama ayri bir is: https://github.com/vercel/swr/discussions/942#discussioncomment-2093364